README.md

```diff
@@ -172,6 +172,24 @@ Additionally, required for tests:
 5) Required packages:
    - liblevel-zero-dev (Linux) or level-zero-sdk (Windows)
 
+#### DevDax memory provider (Linux only)
+
+A memory provider that provides memory from a device DAX (a character device file /dev/daxX.Y).
+It can be used when gigantic memory mappings are needed.
+
+The DevDax memory provider does not support the free operation
+(`umfMemoryProviderFree()` always returns `UMF_RESULT_ERROR_NOT_SUPPORTED`),
+so it should be used with a pool manager that will take over
+the managing of the provided memory - for example the jemalloc pool
+with the `disable_provider_free` parameter set to true.
+
+##### Requirements
+
+1) Linux OS
+2) A character device file /dev/daxX.Y created in the OS.
+3) The `UMF_TESTS_DEVDAX_PATH` environment variable set to the path of the device DAX ("/dev/daxX.Y").
+4) The `UMF_TESTS_DEVDAX_SIZE` environment variable set to the size of the "/dev/daxX.Y" device DAX.
+
```

Comment:
I guess you could mention something like in OS provider:

>The `shm_name` parameter should be a null-terminated string of up to NAME_MAX (i.e., 255) characters none of which are slashes.

Reply:
It is not needed here, since they are read by `getenv()` and there are no such restrictions.

Reply:
hmm I can see you're using `NAME_MAX` in the code here... so there's some restriction...?

Reply:
Right, but `NAME_MAX` is `255` and a `"/dev/daxX.Y"` string will never have such length.